tests/group/group_async_work_group_copy_fp16.cpp

```diff
@@ -23,17 +26,26 @@ class TEST_NAME : public util::test_base {
   /** execute the test
   */
   void run(util::logger &log) override {
-
-    auto queue = util::get_cts_object::queue();
-
-    if (!queue.get_device().has_extension("cl_khr_fp16")) {
-      log.note(
-          "Device does not support half precision floating point operations");
-      return;
+    try {
+      auto queue = util::get_cts_object::queue();
+
+      if (!queue.get_device().has_extension("cl_khr_fp16")) {
+        log.note(
+            "Device does not support half precision floating point operations");
+        return;
+      }
+      // Test using queue constructed already
+      for_type_and_vectors<check_type, cl::sycl::half>(queue, log,
+          "cl::sycl::half");
+      for_type_and_vectors<check_type, cl::sycl::cl_half>(queue, log,
+          "cl::sycl::cl_half");
+
+    } catch (const cl::sycl::exception &e) {
+      log_exception(log, e);
+      cl::sycl::string_class errorMsg =
```

Comment:
Just use normal strings everywhere

Reply:
Yes, but the codebase still uses deprecated SYCL aliases. Wouldn't it be better to replace all of them in a separate PR?

Reply:
Done for all files touched by this change. See f1df84b